README.md

```diff
@@ -61,10 +61,10 @@ steps:
 > **Note**: Due to the peculiarities of YAML parsing, it is recommended to wrap the version in single quotation marks:
 > 
 > ```yaml
->   go-version: '1.20'
+>   go-version: '1.21'
 >  ```
 >  
-> The recommendation is based on the YAML parser's behavior, which interprets non-wrapped values as numbers and, in the case of version 1.20, trims it down to 1.2, which may not be very obvious.
+> The recommendation is based on the YAML parser's behavior, which interprets non-wrapped values as numbers and, in the case of version 1.21, trims it down to 1.2, which may not be very obvious.
```

Comment:
It won't be the case. YAML removes 0 if it is not a string that is why you need to revert these changes.

Reply:
or sorry, this with a global replace, let me revert this. good catch.

Reply:
Hello @chenrui333. It looks like you didn't change it.